tests/factories/contacts/cht/clinic.js

```diff
@@ -0,0 +1,19 @@
+const Factory = require('rosie').Factory;
+
+const parent = { 
+  _id: 'hc1',
+  parent: {
+    _id: 'dist1'
+  } 
+};
+
+Factory.define('clinic')
+  .attr('_id','clinic1')
+  .attr('parent', parent)
+  .attr('type', 'clinic')
+  .attr('is_name_generated', 'true')
+  .attr('name', 'Househould 1')
```

Comment:
Similar to my comment above about the uuid, this value is going to be set the same on all objects generated by this factory. I think you probably need some kind of sequence here. Otherwise you could run into issues where you create 2 objects, do things with them, compare the names, and find them to match, except they're actually 2 different objects.

Please consider that concept and apply it in all factories where you don't want the same value to be used on all objects generated by the factory.